pyk/src/pyk/kllvm/compiler.py

```diff
@@ -91,3 +91,41 @@ def compile_runtime(
 
     assert module_file.is_file()
     return module_file
+
+
+# -------------------------------
+# utility for generation of hints
+# -------------------------------
+
+HINTS_FILE_NAME: Final = 'hints.bin'
```

Comment:
This could be a defaulted parameter to `generate_hints`, rather than a global.

Reply:
Done!